pkg/apis/kops/componentconfig.go

```diff
@@ -475,6 +475,9 @@ type KubeAPIServerConfig struct {
 
 	// EnableProfiling enables profiling via web interface host:port/debug/pprof/
 	EnableProfiling *bool `json:"enableProfiling,omitempty" flag:"profiling"`
+
+	// CorsAllowedOrigins enables CORS for comma separated value
+	CorsAllowedOrigins string `json:"corsAllowedOrigins,omitempty" flag:"cors-allowed-origins"`
```

Comment:
```suggestion
	// CorsAllowedOrigins is a list of origins for CORS. An allowed origin can be a regular
	// expression to support subdomain matching. If this list is empty CORS will not be enabled.
	CorsAllowedOrigins []string `json:"corsAllowedOrigins,omitempty" flag:"cors-allowed-origins"`
```

Reply:
Just want to make: Because the k8s API Server accepts a `stringSlice` this can also be a list and will be automatically converted? According to [API CLI args](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/) it needs to be comma separated 
> List of allowed origins for CORS, comma separated

Reply:
It can be one flag with commas or it can be multiple flags.